etcdserver/api/v3rpc/key.go

```diff
@@ -134,6 +134,14 @@ func (s *kvServer) Compact(ctx context.Context, r *pb.CompactionRequest) (*pb.Co
 	return resp, nil
 }
 
+func (s *kvServer) Hash(ctx context.Context, r *pb.HashRequest) (*pb.HashResponse, error) {
+	resp, err := s.kv.Hash(ctx, r)
+	if err != nil {
+		return nil, togRPCError(err)
```

Comment:
well... i mean `togRPCError` already convert it to internal error if this is an unknown type to the function. Please check the code of `togRPCError`